README.md

```diff
@@ -85,15 +84,13 @@ Applications with support of IPTV streams.
 - [TVirl](https://play.google.com/store/apps/details?id=by.stari4ek.tvirl) - Special Android TV Input Service to integrate IPTV channels to pre-installed system TV App like [Live Channels](https://play.google.com/store/apps/details?id=com.google.android.tv).
 - [CosmiTV Player](https://play.google.com/store/apps/details?id=com.cosmiquest.tv) - IPTV Player and DVR for Android that resembles the cable tv box.
 - [kantv](https://github.com/zhouwg/kantv) - Open source player for Android.
+- [M3UAndroid](https://github.com/realOxy/M3UAndroid) - Open source stream player for Android. This application allows you to play in zapping mode, mark channels as favorites and other media player enhancements.
```

Comment:
- new links should be added to the end of the list
- the description should consist of one sentence